tests/Fixtures/Console/Hello.php

```diff
@@ -23,8 +24,14 @@ public function world(string $input): void
     }
 
     #[ConsoleCommand]
-    public function test(?int $optionalValue = null, bool $flag = false): void
-    {
+    public function test(
+        #[ConsoleArgument]
+        ?int $optionalValue = null,
+        #[ConsoleArgument(
+            name: 'custom-flag',
```

Comment:
Do we already have a test for this? That's still missing as far as I can tell?

Reply:
This seems to be a console command fixture I've updated it to test.
But I didn't find where it is used as test in phpunit  (  but I'm really bad at testing )